src/apps/Altinn.AccessManagement/src/Altinn.AccessMgmt.PersistenceEF/Constants/PackageConstants.cs

```diff
@@ -222,6 +262,14 @@ public static IReadOnlyCollection<TranslationEntry> AllTranslations()
             ProviderId = ProviderConstants.Altinn3,
             AreaId = AreaConstants.AgricultureForestryHuntingFishingAndAquaculture,
         },
+        EN = TranslationEntryList.Create(
+            KeyValuePair.Create("Name", "Hunting and Wildlife Management"),
```

Comment:
ser det er forskjellig bruk av casing på de engelske navnene enn bokmål/nynorsk. Er det med vilje/konsekvent gjort?

Ellers tenker jeg vel at det burde være likt så "Hunting and wildlife management"